API/matchmaking/app.js

```diff
@@ -164,16 +166,44 @@ exports.lambdaHandler = async (event, context) => {
           console.log('Describe Matchmaking Ticket');
           ticket = await describeMatchmaking(ticketParams.TicketId);
           console.log(ticket);
-
           // Respond with HTTP 200 OK and Token
           console.log('200 OK');
-          response = {
-            statusCode: 200,
-            body: JSON.stringify({
-              message: 'Ticket Status Retrieved',
-              Ticket: ticket,
-            }),
-          };
+          if (ticket.Status === 'COMPLETED') {
+            console.log(ticket.GameSessionConnectionInfo.MatchedPlayerSessions);
+            response = {
+              statusCode: 200,
+              body: JSON.stringify({
+                message: 'Ticket Status Retrieved',
+                TicketId: ticket.TickedId,
+                Status: ticket.Status,
+                StatusReason: ticket.StatusReason,
+                StatusMessage: ticket.StatusMessage,
+                StartTime: ticket.StartTime,
+                EndTime: ticket.EndTime,
+                PlayerId: ticket.Players[0].PlayerId,
+                GameSessionArn: ticket.GameSessionConnectionInfo.GameSessionArn,
+                IpAddress: ticket.GameSessionConnectionInfo.IpAddress,
+                DnsName: ticket.GameSessionConnectionInfo.DnsName,
+                Port: `${ticket.GameSessionConnectionInfo.Port}`,
```

Comment:
Do we want to specify which port? Since this is the TCP Port right?

Reply:
Possibly in the future. At the moment AWS only sends the TCP port so there is only ever one port to be sent down to the client. If that changes in the future, we can look at differentiating them then.